cmd/server/assets/codestatus/show.html

```diff
@@ -18,21 +18,24 @@ <h1>Verification code status</h1>
       Status of a verification code previously shared with your patient.
     </p>
 
-    <div class="card mb-3">
+    <div class="card mb-3" style="max-width: 30rem">
       <div class="card-header">
         Code status
       </div>
-      <div class="card-body">
-        <div class="list-group">
+      <div class="list-group">
+        <div class="list-group-item">
           <h5 class="mb-1">UUID</h5>
           <p class="mb-1 text-monospace">{{.uuid}}</p>
         </div>
-        <div class="list-group">
+        <div class="list-group-item">
           <h5 class="mb-1">Status</h5>
           <p class="mb-1">{{.status}}</p>
         </div>
-        <div class="list-group">
-          <h5 class="mb-1">Expiry</h5>
+        <div class="list-group-item">
+          <div class="d-flex w-100 justify-content-between">
+            <h5 class="mb-1">Expiry</h5>
+            <button type="button" id="invalidate" class="btn btn-primary btn-sm">Invalidate code now</button>
```

Comment:
I kinda think this button should be below the form and definitly have the btn-danger class